pom.xml

```diff
@@ -10,7 +10,7 @@
 
 	<groupId>org.openmrs.module</groupId>
 	<artifactId>ugandaemr-iss</artifactId>
-	<version>1.0.0-SNAPSHOT</version>
+	<version>1.0.0</version>
```

Comment:
@ssmusoke isn't this supposed to change with a release instead of just editing

Reply:
@slubwama This is not pushed to the OpenMRS repository - hence editing it manually. It is the same thing